src/main/java/com/blackducksoftware/integration/hub/alert/web/model/distribution/SlackDistributionRestModel.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.alert.web.model.distribution;
+
+import org.apache.commons.lang3.builder.RecursiveToStringStyle;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+public class SlackDistributionRestModel extends CommonDistributionConfigRestModel {
+    private static final long serialVersionUID = -3032738984577328749L;
+
+    private String channelName;
+    private String channelUsername;
+
+    public SlackDistributionRestModel() {
+
+    }
+
+    public SlackDistributionRestModel(final String id, final String channelName, final String channelUsername, final String distributionConfigId, final String distributionType, final String name, final String frequency,
+            final String notificationType, final String filterByProject) {
+        super(id, distributionConfigId, distributionType, name, frequency, notificationType, filterByProject);
+        this.channelName = channelName;
+        this.channelUsername = channelUsername;
+    }
+
+    public static long getSerialversionuid() {
+        return serialVersionUID;
+    }
+
+    public String getChannelName() {
+        return channelName;
+    }
+
+    public String getChannelUsername() {
+        return channelUsername;
+    }
+
+    public void setChannelName(final String channelName) {
+        this.channelName = channelName;
+    }
+
+    public void setChannelUsername(final String channelUsername) {
+        this.channelUsername = channelUsername;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
The superclass has this. We can remove it.

Reply:
+1